target/linux/ramips/image/mt7621.mk

```diff
@@ -9,10 +9,6 @@ DEFAULT_SOC := mt7621
 
 DEVICE_VARS += ELECOM_HWNAME LINKSYS_HWNAME DLINK_HWID
 
-ifdef CONFIG_LINUX_5_10
-  DTS_CPPFLAGS += -DDTS_LEGACY
```

Comment:
This flag is used in "mt7621.dtsi", please clean it as well.

Reply:
thanks!